demo/test.c

```diff
@@ -515,13 +443,127 @@ static int test_mp_invmod(void) {
 
    mp_clear_multi(&a, &b, &c, &d, NULL);
    return EXIT_SUCCESS;
- LBL_ERR:
+LBL_ERR:
    mp_clear_multi(&a, &b, &c, &d, NULL);
    return EXIT_FAILURE;
 
 }
 
-static int test_mp_set_double(void) {
+#include <float.h>
+
+#if ((defined __m68k__) || (defined __MC68K__) || (defined M68000))
+/* VERY simple comparing function, for use in this case and this case only! */
+/* There is such a macro in tommath_private.h which is not include'd in this listing */
+#ifndef MIN
+#define MIN(x, y) (((x) < (y)) ? (x) : (y))
+#endif
+static double s_abs(double s_d)
+{
+   return (s_d < 0.0)?-s_d:s_d;
+}
+static float s_absf(float s_d)
+{
+   return (s_d < 0.0f)?-s_d:s_d;
+}
+static int s_compare_doubles(double s_a, double s_b)
+{
+   double abs_a, abs_b, delta;
+
+   /* NaN, inf's and subnormals ignored, not needed in this case*/
+
+   if (s_a == s_b) {
+      return 1;
+   }
+#ifdef LTM_WARN_X87_EXT_PREC
```

Comment:
@czurnieden this define appears nowhere else what should we do with it?

Reply:
It's a bug: "Functionality not documented."

Didn't want to put it in the makefiles for the rule `test_standalone`, too much output, so I made it optional, as a kind of `--verbose`.

Reply:
OK